### DISCUSSION TITLE
Remove timer from Delaying Page option

### DISCUSSION THREAD

**Discussion by Hanzipanzer** [proginosko/LeechBlockNG#518] [Ideas]
I would prefer it if there was an option to turn off the timer on the delaying page. I feel the timer causes me to focus on that making it less hard to outlast it. I think the uncertainty would help me go do something else besides waiting. 
Waiting without indication can often make time feel like its moving more slowly.
Thanks in advance.

**Comment by proginosko** [comment]
The next release will allow custom style (CSS) for blocking/delaying page, so you can hide/blur the number of seconds if you like.

**Comment by proginosko** [comment]
You can now do this in version 1.6.8. For example, enter the following as the custom style under Appearance in General options:

`.delay { filter: blur(4px); }`
